services/QuillLMS/app/workers/student_learning_sequences/handle_assignment_worker.rb

```diff
@@ -4,8 +4,10 @@ module StudentLearningSequences
   class HandleAssignmentWorker
     include Sidekiq::Worker
 
-    def perform(classroom_unit_id, student_id)
-      HandleAssignment.run(classroom_unit_id, student_id)
+    # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
I guess you could define a separate worker if you don't want the rubocop disable:
```
class BackfillHandleAssignmentWorker 
  include Sidekiq::Worker
  def perform(classroom_unit_id, student_id)
    HandleAssignment.run(classroom_unit_id, student_id, backfill: true)
  end
end
```
but I don't have strong feelings either way.

Reply:
Actually, that's not a bad suggestion.  I'll refactor.